Description: Skip test that started failing with 3.25
 That way we can still benefit by running all the other tests.
Author: Jeremy Bicha <jbicha@ubuntu.com>
Bug: https://bugzilla.gnome.org/785319
---
 clutter/tests/conform/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -26,7 +26,6 @@ actor_tests = \
 	actor-layout \
 	actor-meta \
 	actor-offscreen-limit-max-size \
-	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-size \
 	$(NULL)
